### PR TITLE
Barbieri1982 week43

### PR DESCRIPTION
**Pull request by barbieri1982**
several documentation corrections/improvements

### CODE REVIEW

File: pages/web/hosting/diagnostic_fix_500_internal_server_error/guide.en-asia.md [hunk 1]
```diff
@@ -78,7 +78,7 @@ The [FileZilla guide](../web_hosting_filezilla_user_guide/#file-and-folder-permi
 
 For security reasons, your website hides all technical details about the error 500.
 
-To verify these details, you can use a [SSH connection](../web_hosting_ssh_on_web_hosting_packages/). (SSH is only available with a hosting plan of the [Business or Developer range](https://www.ovhcloud.com/asia/web-hosting/).)
+To verify these details, you can use a [SSH connection](../web_hosting_ssh_on_web_hosting_packages/). (SSH is only available with the [Professional Hosting, Performance Hosting and Cloud Web Hosting](https://www.ovhcloud.com/asia/web-hosting/).)
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/diagnostic_fix_500_internal_server_error/guide.en-asia.md]: @Kazarus4 , please remove Cloud Web mentions on CA/APAC subs.

File: pages/web/hosting/exporter-son-site-web/guide.en-asia.md [hunk 2]
```diff
@@ -62,6 +62,8 @@ To identify the folder that the website is stored on, go to the `Multi-site`{.ac
 To retrieve a backup of your database, please read our guide on
 [Retrieving the backup of a Web Hosting plan’s database](../web_hosting_database_export_guide/).
 
+If you are using a **CloudDB** database for your website, you can go to the ‘Databases’ tab, click the `...`{.action} button to the right of the database, and click ‘Back up now’ in the drop-down menu.
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/exporter-son-site-web/guide.en-asia.md]: @Kazarus4 , CloudDB is not offered outside Europe. Please remove it from all CA/APAC subs.

File: pages/web/hosting/multisites_configure_multisite/guide.de-de.md [hunk 3]
```diff
@@ -105,7 +110,7 @@ Nachdem die Domain hinzugefügt wurde, gehen Sie zu [Schritt 3: Ihre Website onl
...
+> |TXT|Tab `Multisite`{.action}, klicken Sie auf `Konfiguration des Tokens ovhcontrol`{.action}|OVHcloud stellt sicher, dass das Hinzufügen jeder externen Domain legitim ist. Achten Sie darauf, den TXT-Eintrag mit der Subdomain ovhcontrol (z. B. ovhcontrol.mydomain.ovh) in der DNS-Zone zu erstellen, die für die hinzuzufügende Domain zuständig ist.<br></br>Bitte beachten Sie, dass Sie, wenn Sie `blog.mydomain.ovh` hinzufügen möchten, den Eintrag für die Subdomain `ovhcontrol.mydomain.ovh` und nicht `ovhcontrol.blog.mydomain.ovh` erstellen müssen.<br></br> Um zu dieser zu gelangen, finden Sie die [DNS Server](../../domains/webhosting_allgemeine_informationen_zu_den_dns_servern/#dns-konzept-verstehen), mit denen Ihre Domain verbunden ist. Sie müssen nur die Hauptdomain validieren, nicht alle Subdomains.|
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/multisites_configure_multisite/guide.de-de.md]: @tcpdumpfbacke , please review DE editions

File: pages/web/hosting/multisites_configure_multisite/guide.fr-ca.md [hunk 4]
```diff
@@ -147,7 +152,7 @@ A contrario, si vous souhaitez installer manuellement votre site, munissez-vous
 >
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/multisites_configure_multisite/guide.fr-ca.md:152]: @Kazarus4, edition of line 113 is missing on FR subs

**Review by OVH-Team-Guides — CHANGES_REQUESTED** [review]

File: pages/web/hosting/diagnostic_fix_500_internal_server_error/guide.en-asia.md (hunk 1 above)
Comment (Kazarus4) [reply pages/web/hosting/diagnostic_fix_500_internal_server_error/guide.en-asia.md]: done

File: pages/web/hosting/exporter-son-site-web/guide.en-asia.md (hunk 2 above)
Comment (Kazarus4) [reply pages/web/hosting/exporter-son-site-web/guide.en-asia.md]: done

File: pages/web/hosting/multisites_configure_multisite/guide.fr-ca.md (hunk 4 above)
Comment (Kazarus4) [reply pages/web/hosting/multisites_configure_multisite/guide.fr-ca.md:152]: I don't understand this point

Comment (OVH-Team-Guides) [reply pages/web/hosting/multisites_configure_multisite/guide.fr-ca.md:152]: Sorry, line 109

Comment (Kazarus4) [reply pages/web/hosting/multisites_configure_multisite/guide.fr-ca.md:152]: done

**Review by OVH-Team-Guides — APPROVED** [review]

**Comment by OVH-Team-Guides** [comment]
Thank you for your contribution @barbieri1982

**Pull request merged**